### ISSUE TITLE
Some members are displayed twice in member list 

### ISSUE THREAD

**Issue by marthijnvdn** [svthalia/Reaxit#359]
### Describe the bug
When scrolling through the Member list, some members were displayed twice. When reloading the list, different members were displayed twice. This happened both times to people that started with the same letter.

### How to reproduce
1. Go to Member list in the app
2. Scroll down until you (hopefully) find a duplicate

### Expected behaviour
No duplicates

### Screenshots
![IMG-20230228-WA0037](https://user-images.githubusercontent.com/27356773/221906832-b14e03e5-ece4-4893-b021-367582e119ef.jpg)

### Environment:

 - Device: Samsung S320
 - OS: Android 14
 - Version 3.3.1

### Additional context
Idk how to replicate the bug, and I really don't know what could possibly cause this, as it happens to different people each time

**Comment by DeD1rk** [comment]
@JAicewizard could it be missing keys? Sounds like a small easily fixable thing so can we include it in 3.4.0?

**Comment by JAicewizard** [comment]
I just checked and these people are not duplicate to me, so unable to reproduce but we did also have this as an issue earlier today. It is difficult to fix when its not reproducable, the best we can do is try to filter duplicate values?
I also checked that there arn't any missing members in the screenshot.
The last 2 versions of evi and ewout seem to be the correct one according to my phone, maybe the first are from an old cached version and the later ones are after new members joined?
